kube/src/test/java/org/openshift/ping/kube/test/SimplePingTest.java

```diff
@@ -18,13 +18,15 @@
 
 import org.jgroups.protocols.FILE_PING;
 import org.jgroups.stack.Protocol;
+import org.junit.Ignore;
 
 /**
  * A parallel test to ZK tests.
  * Just so we know tests are fine.
  *
  * @author <a href="mailto:ales.justin@jboss.org">Ales Justin</a>
  */
+@Ignore("To be clarified with Bela")
```

Comment:
This is a regression between 4.0.0.Beta3 and 4.0.0.CR1. I created [1] to fix it. A workaround is decsribed in [1].

Fixed in 4.0.1.

[1] https://issues.jboss.org/browse/JGRP-2159